package/boot/uboot-mediatek/patches/437-add-cmcc_rax3000m.patch

```diff
@@ -638,11 +638,12 @@
 +_bootmenu_update_title=setenv _bootmenu_update_title ; setenv bootmenu_title "$bootmenu_title       [33m$ver[0m"
 --- /dev/null
 +++ b/cmcc_rax3000m-nand_env
-@@ -0,0 +1,56 @@
+@@ -0,0 +1,57 @@
 +ipaddr=192.168.1.1
 +serverip=192.168.1.254
 +loadaddr=0x46000000
 +console=earlycon=uart8250,mmio32,0x11002000 console=ttyS0
++bootargs=root=/dev/fit0 rootwait
```

Comment:
Hi! Since there's no bootargs for nand variant in the first place, I think we can omit this line and use bootargs in device-tree directly. This way we don't need to update bootloader every time we want a cmdline change.

Reply:
tbh I'd prefer keep it consistent with the dts, and (hopefully) `bootargs-override` hack can be dropped from dts some day.

Reply:
I agree but at least it will land on new installation.

Reply:
My point is: This bootloader change for the nand variant is unnecessary in the first place and only puts in new roadblocks for future cmdline changes (i.e. we have to use bootargs-override hack instead of the usual bootargs).
For most devices in OpenWrt, either bootloader does not supply a cmdline or we use bootargs-override to completely ignore the bootloader-supplied cmdline.

Reply:
>  puts in new roadblocks for future cmdline changes

The new FIT format has been upstreamed and won't change for quite some time.

> For most devices in OpenWrt, either bootloader does not supply a cmdline

Because these bootloader-s are not made by OpenWrt ;)
For our self-compiled bootloader, the situation seems to be different.

However I removed it to get this PR merged faster.